modular_zubbers/code/datums/ert.dm

```diff
@@ -1,8 +1,34 @@
 /*
 *	BUBBER MODULAR OUTFITS FILE
 *	PUT ANY NEW ERT OUTFITS HERE
+*	OR CHANGES TO ERT HERE
 */
 
+
+/datum/ert
+	code = "Green"
+
+/datum/ert/marine
+	code = "Military Green"
+
+/datum/ert/centcom_official
+	code = "Gold"
+
+/datum/ert/inquisition
+	code = "Spanish Red"
+
+/datum/ert/janitor
+	code = "Purple"
+
+/datum/ert/intern
+	code = "Uhhhhhh"
```

Comment:
```suggestion
	code = "Unpaid"
```

Not necissarily a requirement, I just think it would be funny. Otherwise the PR is approved